tailwind.config.js

```diff
@@ -16,6 +16,9 @@ module.exports = {
     defaultExtractor: content => content.match(/[A-Za-z0-9-_:/]+/g) || [],
   },
   theme: {
+    maxWidth: {
+      'full': '99%',
```

Comment:
Why do we need this?

Reply:
Was going for a temporary hack to fix width issue. Removed it and fixed it properly now.